Makefile

```diff
@@ -1,25 +1,29 @@
 .DEFAULT_GOAL := help
 SHELL=/bin/bash
-SYMFONY=cd tests/Application && symfony
+APP_DIR=tests/Application
+SYLIUS_VERSION=1.10
+SYMFONY=cd ${APP_DIR} && symfony
 COMPOSER=symfony composer
 CONSOLE=${SYMFONY} console
 export COMPOSE_PROJECT_NAME=coliship
+PLUGIN_NAME=sylius-${COMPOSE_PROJECT_NAME}-plugin
 COMPOSE=docker-compose
-YARN=cd tests/Application && yarn
+YARN=yarn
 
 ###
 ### DEVELOPMENT
 ### ¯¯¯¯¯¯¯¯¯¯¯
 
-install: platform sylius ## Install the plugin
+install: application platform sylius ## Install the plugin
 .PHONY: install
 
 up: docker.up server.start ## Up the project (start docker, start symfony server)
 stop: server.stop docker.stop ## Stop the project (stop docker, stop symfony server)
 down: server.stop docker.down ## Down the project (removes docker containers, stop symfony server)
 
 reset: docker.down ## Stop docker and remove dependencies
```

Comment:
If you don't have the test application then the docker.down will fail so will the reset.